File: spring-java/src/main/java/com/jmpark/springjava/entity/TicketEntity.java
package com.jmpark.springjava.entity;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@RequiredArgsConstructor
@Getter
@ToString
@EqualsAndHashCode(callSuper = false)
public class TicketEntity extends BaseTimeEntity {
    @Id
    String id = "";

    Integer areaId = null;
    String userId = null;
    String type = null;
    Integer minute = 0;
    Integer price = 0;
    String status = null;
    String payMethod = null;
    String payDate = null;
    String cardCode = null;
    String cardNumber = null;
}

File: spring-kotlin/src/main/kotlin/com/jmpark/springkotlin/entity/TicketEntity.kt
package com.jmpark.springkotlin.entity

import javax.persistence.Entity
import javax.persistence.Id

@Entity
data class TicketEntity (
    @Id
    val id : String = "",

    var areaId: Int? = null,
    var userId: String? = null,
    var type: String? = null,
    var minute: Int? = 0,
    var price: Int? = 0,
    var status: String? = null,
    var payMethod: String? = null,
    var payDate: String? = null,
    var cardCode: String? = null,
    var cardNumber: String? = null
) : BaseTimeEntity()